node/light_test.go

```diff
@@ -62,3 +62,16 @@ func TestNewLightWithHost(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, node.Host.ID(), nw.Peers()[0])
 }
+
+func TestLight_WithBootstrapPeers(t *testing.T) {
```

Comment:
Why Light is allowed to have tests only for Bootstrapp and not mutual? :sad_boy_cry_png:

Reply:
>_< Ok 🙈